build/wix/main.wxs

```diff
@@ -157,6 +157,18 @@
       <ComponentRef Id="PathUser" />
       <ComponentRef Id="PathSystem" />
     </Feature>
+    <UI>
+       <UIRef Id="WixUI_RD" />
+       <Publish Dialog="ExitDialog"
+         Control="Finish"
+         Event="DoAction"
+         Value="LaunchApplication"
+       >WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT = 1 and NOT Installed</Publish>
+    </UI>
+    <Property Id="WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT" Value="Run Rancher Desktop" />
+    <Property Id="WixShellExecTarget" Value="[#myapplication.exe]" />
+    <CustomAction Id="LaunchApplication" BinaryKey="WixCA" DllEntry="WixShellExec" Impersonate="yes" />
```

Comment:
We already have a `<CustomAction Id="RunApplication">`; do we need a separate one here? (It's possible that we need it if for some reason we can't share it with running it after an update, but it'd be good to check…)
https://github.com/rancher-sandbox/rancher-desktop/blob/0a26306847d0afedb2f0c98cff02c21149159892/build/wix/main.wxs#L141-L148

Reply:
I'm not sure this `RunApplication` is ever hit. If I replace the ref (i.e. `Value`) in the `ExitDialog` dialog from `LaunchApplication` to `RunApplication` I get error 2762, which seems to be a catch-all dialog for installer actions.

Reply:
Argh, yeah, couldn't get this to work, because it seems like `<CustomAction FileKey="">` is meant for the server side (that is, the privileged section of the code), so by the time the exit dialog shows up it's too late (and it always errors out when trying to run the action).  We can't invert it to always use `WixShellExec` either, because the opposite problem happens :(